extra/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+test -e /etc/motioneye/motioneye.conf || \
+cp /usr/share/motioneye/extra/motioneye.conf.sample /etc/motioneye/motioneye.conf ; \
```

Comment:
This does not work anymore. Shipping files like this is deprecated with setuptools, they are now shipped as part of the Python package itself. Hmm, I think best is to use the new `motioneye_init` command here as well. It also includes all non-Python dependencies, setting up default config and systemd unit. Okay the letter is not required in Docker, and not available, I guess, so we'd need to add a CLI option to skip systemd setup.